deployments/moonriver.json

```diff
@@ -29,5 +29,6 @@
   "TokenWrapper": "0x5215E9fd223BC909083fbdB2860213873046e45d",
   "GenericSwapFacetV3": "0x31a9b1835864706Af10103b31Ea2b79bdb995F5F",
   "LiFiDEXAggregator": "0x6140b987d6B51Fd75b66C3B07733Beb5167c42fc",
-  "EmergencyPauseFacet": "0x6F2baA7cd5F156CA1B132F7FF11E0fa2aD775F61"
+  "EmergencyPauseFacet": "0x6F2baA7cd5F156CA1B132F7FF11E0fa2aD775F61",
+  "Permit2Proxy": "0x89c6340B1a1f4b25D36cd8B063D49045caF3f818"
```

Comment:
no contract at this address: https://moonriver.moonscan.io/address/0x89c6340B1a1f4b25D36cd8B063D49045caF3f818

Reply:
not true, the block explorer is just not showing it. If you run cast code you'll see the bytecode

Reply:
in the deployment_logs file the Permit2Proxy VERFIED field is false. all good

> If you run cast code you'll see the bytecode

all good